TelepresenceBot/server-unit/core/public/js/connect.js

```diff
@@ -0,0 +1,24 @@
+var intervalId;
+
+$(document).ready(function(){
+
+    $("input").mousedown(function(){
+        console.log("mouseDown");
+        var message = $(this).val();
+        intervalId = setInterval(function() {
+            sendMessage(message);
+        }, 500);
```

Comment:
❗️ We should match the `COMMAND_REPEAT_DELAY` defined both in the Android and Arduino parts.
In that case it's 100ms